### PR TITLE
Update CI doc based on #58

### PR DESCRIPTION
**Pull request by fmuyassarov**
Update CI doc based on [#58](https://github.com/metal3-io/project-infra/pull/58).

### CODE REVIEW

File: jenkins/README.md [hunk 1]
```diff
@@ -45,6 +45,15 @@ If the author is not in the whitelist but should be trusted then by adding a
 comment **add to whitelist** on the PR, the author will then be able to run the
 jobs on its own.
 
+### How to clean up leftover VMs from CityCloud
+There is a Jenkins [master job](https://jenkins.nordix.org/view/Airship/job/airship_master_integration_tests_cleanup/)
+that every 6 hours cleans up all the leftover VMs, which failed to be deleted
```
Comment (kashifest) [inline jenkins/README.md]: fails to be ?

File: jenkins/README.md [hunk 2]
```diff
@@ -45,6 +45,15 @@ If the author is not in the whitelist but should be trusted then by adding a
...
 jobs on its own.
 
+### How to clean up leftover VMs from CityCloud
+There is a Jenkins [master job](https://jenkins.nordix.org/view/Airship/job/airship_master_integration_tests_cleanup/)
+that every 6 hours cleans up all the leftover VMs, which fail to be deleted
+at the end of v1alphaX integration test.
+
+**Note:** If you want to trigger cleaning job manually, you can use the
+`/test-clean` phrase within an open pull request under
+[metal3-io/project-infra](https://github.com/metal3-io/project-infra) repo.
```
Comment (maelk) [inline jenkins/README.md]: The purpose is also to be able to test a change in the cleaning script. To run it manually, I would rather recommend to do it through the Jenkins Web UI

Comment (fmuyassarov) [reply jenkins/README.md]: Does everyone have an access to the Jenkins UI to do so? or should I skip this part here?

File: jenkins/README.md [hunk 3]
```diff
@@ -45,6 +45,13 @@ If the author is not in the whitelist but should be trusted then by adding a
 comment **add to whitelist** on the PR, the author will then be able to run the
 jobs on its own.
 
+### How to clean up leftover VMs
```
Comment (maelk) [inline jenkins/README.md]: This title is not exactly reflecting the content. Maybe "Cloud Resources cleanup" ?

Comment (fmuyassarov) [reply jenkins/README.md]: done

**Pull request merged**